common-utils/src/main/java/com/common/ledger/utils/LedgerResponse.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * 
+ */
+package com.common.ledger.utils;
+
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_CODE_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_DATA_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_ERRORCODE_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_MESSAGE_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_PATH_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_STATUS_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_TIMESTAMP_KEY;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+
+public class LedgerResponse implements Serializable {
+
+	private static final long serialVersionUID = 8218105114689710526L;
+
+	private transient Map<String, Object> responseMap;
+	HttpStatus httpStatus;
+	
+	/**
+	 * 
+	 */
+	public LedgerResponse() {
+		responseMap = new HashMap<>(5);
+	}
+	/**
+	 * 
+	 * @param code
+	 * @param message
+	 * @param result
+	 */
+	public LedgerResponse(HttpStatus code , String message , Object result) {
+		responseMap = new HashMap<>(5);
+		setHttpStatus(code);
+		setMessage(message);
+		if(result != null) {
+			setData(result);
+		}
```

Comment:
Could u please explain , we are passing Object and setData method is taking String

Reply:
we just trying to maintain some standard response in all services so we are using this util. code and message we most of the time send as a response. if any additional data we required to send we can push in setData method